tests/test_custom_models.py

```diff
@@ -942,8 +956,8 @@ def test_parameters_after_loading_model(self, test_name, model_id, config_cls, c
     @parameterized.expand(TEST_CASES)
     def test_disable_adapters(self, test_name, model_id, config_cls, config_kwargs):
         X = self.prepare_inputs_for_testing()
-        model = self.transformers_class.from_pretrained(model_id).to(self.torch_device).eval()
-
+        model = self.transformers_class.from_pretrained(model_id).to(self.torch_device)
+        model.eval()
```

Comment:
Why this change?

Reply:
This is a mistake. It should work the way it was. Will get to it.